### DISCUSSION TITLE
Checking visibility state of ar-button

### DISCUSSION THREAD

**Discussion by rxptr** [google/model-viewer#2833] [Q&A] [answered]
How can I check if my custom `ar-button` is displayed or not ?
I want be able to display another button in case that AR Support is not detected and it's button is absent.

**Comment by ghost** [comment, marked as answer]
you can check `modelViewer.canActivateAR` to determine if AR is supported.

**Reply by rxptr** [reply]
Hey Tim, thank you for the answer. I had some difficulties using `canActivateAR`. I did some digging and found this discussion [#2377](https://github.com/google/model-viewer/discussions/2377) that further cleared things up for me.
